python/ray/dashboard/client/src/theme.ts

```diff
@@ -117,11 +118,13 @@ export const lightTheme = createTheme(basicTheme, {
       paper: "#fff",
       default: blueGrey[50],
     },
+    divider: "#D2DCE6",
   },
 });
 
 export const darkTheme = createTheme(basicTheme, {
```

Comment:
interesting, didn't realize we already had a palette for dark mode...

@scottsun94, ptal at the design and colors